翻译光源 (#1821)
cocos-creator/creator-docs: en/concepts/scene/light/lightType/spot-light.md

> # Spotlights

Reviewer: ```suggestion
# Spotlight
```

Author: In Unity's documentation, they use plural in the title of all types of light except for ambient light.

A **Spotlight** is a cone-shaped beam of light emitted from a point in one direction, similar to the light produced by a flashlight or stage lighting. Compared to other lights, spotlights have an additional `SpotAngle` property, which is used to adjust the light range of the spotlight.

![spotlight](spotlight/spot-light.jpg)

> In the editor, you can clearly see the position, color and light range of the light as well as its spot angle, as shown in the following image. With the [Transform Gizmos](../../../../editor/toolbar/index.md) in the upper left corner of the editor, you can adjust the position and direction of the spotlight.

Reviewer: ```suggestion
In the editor, one can clearly see the position, color and light range of the light as well as its spot angle, as shown in the following image. With the [Transform Gizmos](../../../../editor/toolbar/index.md) in the upper left corner of the editor, it is possible to adjust the position and direction of the spotlight.
```

![spotlight](spotlight/spot-light-scene.jpg)

See [Adding A Light](index.md#adding-a-light) for how to add a spotlight to a scene.

For the Spotlight component interface, please refer to [SpotLight API](__APIDOC__/en/classes/component_light.spotlight.html).

## Spotlight Properties

![image](spotlight/spot-light-prop.png)

| Property | Description |
| :------ | :--- |
| Color | Sets the color of the light. |

> | UseColorTemperature | Enables or disables the color temperature. |

Author: ```suggestion
| UseColorTemperature | Whether to enable the color temperature. |
```
